File: Assets/Script/Player.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;


namespace Corona
{
    public class Player : MonoBehaviour
    {

        public int index;
        public string name;
        public bool isLocalPlayer;
        public bool isAdmin;

        // Start is called before the first frame update
        void Start()
        {
            DontDestroyOnLoad(this);

        }

        // Update is called once per frame
        void Update()
        {

        }


    }
}

File: Server/GameObject/Player.js
const Position = require("./Position");
const Rotation = require("./Rotation");

class Player {

  constructor (name,index,isAdmin){
    this.name = name;
    this.position = new Position(0,0,0);
    this.rotation = new Rotation(0,0,0);
    this.index = index;
    this.isAdmin = isAdmin
  }

}
module.exports = Player
